packages/core/test/acceptance/control_flow_exploration_spec.ts

```diff
@@ -243,4 +244,226 @@ describe('control flow', () => {
       expect(fixture.nativeElement.textContent).toBe('default ');
     });
   });
+
+  describe('for', () => {
+    it('should create, remove and move views corresponding to items in a collection', () => {
+      function App_ng_template_0_Template(rf: number, ctx: any) {
+        if (rf & 1) {
+          ɵɵtext(0);
+        }
+        if (rf & 2) {
+          const item = ctx.$implicit;
+          const idx = ctx.$index;
+          ɵɵtextInterpolate2('', item, '(', idx, ')|');
+        }
+      }
+
+      class TestComponent {
+        items = [1, 2, 3];
+
+        static ɵcmp = ɵɵdefineComponent({
+          type: TestComponent,
+          selectors: [['some-cmp']],
+          decls: 2,
+          vars: 0,
+
+          // {#for (item of items); track item; let idx = index}{{item}}|{/for}
```

Comment:
```suggestion
          // {#for (item of items); track item; let idx = index}{{item}}({{idx}})|{/for}
```

Same in the other tests

Reply:
Thnx!